Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/ParentPass.h

```diff
@@ -58,6 +58,11 @@ namespace AZ
 
             // --- Children related functions ---
 
+            //! If this parent pass is supposed to merge child passes as subpasses
+            //! makes sure @child is a RasterPass and marks the child passes as mergeable.
```

Comment:
What marks the child passes as mergeable? Not clear from comment whether ParentPass needs to mark the child as mergeable or the child needs to mark itself as mergeable.

Reply:
Added clarification comment:
```
            //! When PassData::m_mergeChildrenAsSubpasses is true, the ParentPass will try to merge child passes as subpasses.
            //! Child passes do not mark themselves as mergeable, instead the ParentPass marks its children as mergeable by
            //! calling this function. 
            //! This function makes sure @child is a RasterPass and marks the child passes as mergeable.
            //! if the child is NOT a RasterPass, this ParentPass clears its flag that merges children passes.
```